pom.xml

```diff
@@ -33,13 +33,23 @@
         <updatePolicy>daily</updatePolicy>
       </snapshots>
     </repository>
+
+    <repository>
+      <id>mvn-central</id>
+      <name>Maven Central</name>
+      <url>http://central.maven.org/maven2/</url>
+    </repository>
   </repositories>
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <maven.compiler.source>1.8</maven.compiler.source>
     <maven.compiler.target>1.8</maven.compiler.target>
-    <wildfly.home>../local/appserver/wildfly</wildfly.home>
+    <wildfly.version>10.1.0.Final</wildfly.version>
```

Comment:
version properties should be in the form version.groupId, see 5 lines below for examples